src/Models/VectorStructInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Qdrant\Models;
+
+interface VectorStructInterface
+{
+    /**
+     * Get the name of the vector
+     *
+     * @return string
+     */
+    public function getName(): string;
```

Comment:
While implementing `getName` for `MultiVectorStruct`, we need to return an array for a name or just only the first element of the array. And the result of this function won't be consistent. What do you think about removing this function from interface?

Reply:
We need the interface to support returning just a string in some places. Here, for example: https://github.com/hkulekci/qdrant-php/pull/14/files#diff-23d49b8f87680472348ef6a2b3a63f5becaef507b78a400e4b4a9e8455762ae8R88

As far as I recall, Qdrant only supports searching a single vector field at a time. So in this case, we'll always either use the first vector, or the one the user has passed using `name`.

We could add a separate function, `getNames`, to `MultiVectorStruct`, but I'm not sure where it'd be used.

Reply:
Lets keep as is.